server/pipeline/create.go

```diff
@@ -16,14 +16,16 @@ package pipeline
 
 import (
 	"context"
+	"errors"
 	"fmt"
 	"regexp"
 
 	"github.com/rs/zerolog/log"
 
-	"go.woodpecker-ci.org/woodpecker/v2/pipeline/errors"
+	pipelineErrors "go.woodpecker-ci.org/woodpecker/v2/pipeline/errors"
```

Comment:
We use already `pipeline_errors` in multiple other locations? Would you prefer `pipelineErrors`?